src/main/resources/youtube/branding/xisr_yellow/monochrome/drawable/adaptive_monochrome_ic_youtube_launcher.xml

```diff
@@ -1,18 +1,18 @@
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="108dp"
-    android:height="108dp"
-    android:viewportWidth="108"
-    android:viewportHeight="108">
-  <path
-      android:pathData="M31.11,48.3C31.26,43.28 34.46,38.94 39.42,38.11C43.04,37.51 47.9,37 54,37C60.1,37 64.96,37.51 68.58,38.11C73.54,38.94 76.74,43.28 76.89,48.3C76.96,50.26 77,52.28 77,54C77,55.72 76.96,57.74 76.89,59.7C76.74,64.72 73.54,69.06 68.58,69.89C64.96,70.49 60.1,71 54,71C47.9,71 43.04,70.49 39.42,69.89C34.46,69.06 31.26,64.72 31.11,59.7C31.05,57.74 31,55.72 31,54C31,52.28 31.05,50.26 31.11,48.3Z"
-      android:strokeWidth="5"
-      android:fillColor="#00000000"
-      android:strokeColor="#000000"/>
-  <path
-      android:pathData="M49,48.77C49,45.1 52.63,42.81 55.54,44.65L63.82,49.87C66.73,51.71 66.73,56.29 63.82,58.13L55.54,63.35C52.63,65.19 49,62.9 49,59.23L49,48.77Z"
-      android:fillColor="#000000"
-      android:fillAlpha="0.5"/>
-  <path
-      android:pathData="M42,50.34C42,47.77 44.56,46.17 46.62,47.45L52.46,51.11C54.51,52.39 54.51,55.61 52.46,56.89L46.62,60.55C44.56,61.83 42,60.23 42,57.66L42,50.34Z"
-      android:fillColor="#000000"/>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
adding this is intended? seems unnecessary since it's a vector that will be nested during compile time

Reply:
let it be, so i can merge without waiting

Reply:
Meaning?

Reply:
I meant no need to change anything